src/model/image_info.js

```diff
@@ -321,6 +321,17 @@ export default class ImageInfo {
      * @memberof ImageInfo
      */
     requestImgRDef(callback = null) {
+        if (callback === null)
+            callback = (rdef) => {
+                if (rdef === null || typeof rdef.c !== 'string') return;
```

Comment:
ok, that's good to know that there is an additional safeguard.

the front-end logic does control the disabled state of the button though which I think is helpful so that the user does not think that paste makes sense and then waits for something to happen.